lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -732,57 +733,98 @@ def _get_default_parameters(kymo, channel):
 
     return {
         "pixel_threshold": KymotrackerParameter(
-            "Threshold",
-            "Set the pixel threshold.",
+            "Minimum intensity",
+            "Set the intensity threshold.",
             "int",
             max(1, int(np.percentile(data.flatten(), 98))),
             *(1, max(2, np.max(data))),
             True,
+            r"Minimum pixel intensity defines how many photons are required in a single pixel for "
+            r"something to be considered part of a track. This parameter should be chosen slightly "
+            r"above the baseline photon count. Higher values reject more noise, but actual peaks "
+            r"that should be part of the track may be missed.",
```

Comment:
One thing that has come up in discussions with the HW team is that the signal we observe technically isn't the true number of photons. This is because the detector efficiency also plays a role, so for complete correctness we should refer to these as `"counts"`. I think `"photon counts"` is still accurate also. 

Also in general, I don't think we should expect for users to understand that underlying all this there is a peak finding algorithm, so I'd rather avoid that term in these descriptions.

So maybe just a minor change:
```suggestion
            r"Minimum pixel intensity defines the minimum signal required in a single pixel for "
            r"it to be considered part of a track. This parameter should be chosen slightly "
            r"above the baseline photon count. Higher values reject more noise, but actual points "
            r"that should be part of the track may be missed.",
```

Reply:
👍